elfi/methods/methods.py

```diff
@@ -725,14 +725,15 @@ def __init__(self, model, target=None, outputs=None, batch_size=1, n_acq=150,
             Only needed if model is an ElfiModel
         target_model : GPyRegression, optional
         acquisition_method : Acquisition, optional
+            Method of acquiring evidence points. Defaults to LCBSC.
         bounds : list
             The region where to estimate the posterior for each parameter in
             model.parameters.
             `[(lower, upper), ... ]`
         initial_evidence : int, dict
             Number of initial evidence or a precomputed batch dict containing parameter 
             and discrepancy values
-        n_evidence : int
+        n_acq: int
```

Comment:
I noticed I had some inconsistencies here with `n_evidence` and `n_acq`. I wonder if we should only use `n_evidence`, it would be easy for the user to know immediately how many evidence points he will have in the end (handy for estimating the required time to run the experiment). Also the `initial_evidence` could then be None by default and then be set by the heuristic depending on the dimensions rather than being a fixed quantity.